chat/Makefile.linux

```diff
@@ -18,7 +18,7 @@ INSTALL= install
 all:	chat
 
 chat:	chat.o
```

Comment:
can you try by removing the line below and leave only `chat:	chat.o`?

Reply:
Did you mean to remove "all: chat"? I think somebody could call it with 'make all' and it seems to be used in other makefiles across this project. Also I think such change is not related to the purpose of this PR.

Reply:
No, I mean the whole `$(CC) $(LDFLAGS) -o chat chat.o` and the `chat:` target.
Make has some implicit rules which should be exactly the same

I didn't try but it should work without the target at all

Reply:
Yes it works and also the rule for the chat.o can be removed. But this is not "let's write the shortest possible Makefile" contest. There are more Makefiles in this project which could be simplified with help of implicit rules. I think it should be changed consistently in all Makefiles or nowhere (personally I prefer explicit rules because it's more clear, but it's OT with regard to the original issue).